src/nose2_contrib/jira/callbacks.py

```diff
@@ -72,19 +78,24 @@ def do_nothing(jira_plugin, jira_issue, test, *_):
     jira_plugin.logger.info("did nothing for %(jira_issue)s and test %(test)s", jira_issue=jira_issue.id, test=test)
 
 
-@JiraRegistry.register('warn_regression', False)
-def warn_regression(jira_plugin, jira_issue, test, message):
+def warn_regression(jira_plugin, jira_issue, test, message, *, message_format):
     """
-    Send a message to mark a regression.
+    Send a message to mark a regression. And add it to the ``jira_plugin.regressions`` list.
```

Comment:
ok...maybe regression != testrun?

Reply:
I will definitely define what is a regression.

Reply:
So it seems like you have this thing where the JIRA plugin is storing a list of tests that have specified combination of JIRA status and test status and giving it back to you in a markdown file. This seems potentially more general than what you describe as "regression".  For example, if I'm understanding properly, this could be used to produce a list of known issues where the test is still failing.  With this in mind, it would be ideal to give this a more generic name than "warn regression" (eg store status) and maybe even support multiple lists not just one?

Then allow the user to configure what to do with this list at the end. 

Also, comment has it's own method, so why is it here afterwards?

Reply:
> Then allow the user to configure what to do with this list at the end.

I think I will make such an change in a future version. For now, I do prefer keep as is.

Reply:
About the fact I have the "add_comment" called : JiraPlugin was imagined to focus on reporting regression and feed the jira history. So I think a function that enqueue regression for further reporting and add a comment to jira if the test analysts configured it to do so is what we want.